# SpatialAutocorrelation
Software for assessing whether microbial taxa, genes, and traits have structured spatial distributions.

This software checks whether the taxa, genes, and traits in a set of geographically distributed metagenomic samples are distributed randomly, or if they are aggregated or segregated (i.e., are autocorrelated). Starting with an OTU, gene, or trait table in HDF5 BIOM format (http://biom-format.org) and metadata on the locations where samples were collected, the software calculates the Moran's I statistic for each taxon, gene, or trait, and the statistical significance of that Moran's I statistic using a randomization test. The randomization test uses a Markov chain Monte Carlo algorithm and is highly optimized to allow large data tables to be processed. The shell script SpatialAutocorrelationTest.sh runs an example data set and check for errors with the installation. Addition documentation on the software is available in the file doc/index.html. All source code is located the 'Autocorrelation' and 'Shared' directories at https://github.com/jladau/Source.
